### ISSUE TITLE
(global) remove &lt; and &gt;

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-greekLit#238]
replace with appropriate semantic markup such as `<add>`

**Comment by lcerrato** [comment]
please add any affected files to this report

**Comment by lcerrato** [comment]
NB Although we use `<del>` and `<add>` throughout current editions, these seem to be described in EpiDoc as corrections made by an "ancient hand".  In many cases, these are later editorial interventions, which would suggest other markup.
per TEI "del indicates a deletion present in the source being transcribed, which states the author's or a later scribe's intent to cancel or remove text. surplus indicates material present in the source being transcribed which should have been so deleted, but which is not in fact."
so `<surplus>` and `<supplied>` seem to be preferred for our types of editors but we are headed in the app crit for many of these issues such as #244 ...

**Comment by PonteIneptique** [comment]
@lcerrato The issue here is that Epidoc is originally used for inscriptions and we are using it for modern edition transcriptions. If you look at http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-add.html it says also an annotator or corrector, which would be my understanding of what `<a>` and `[b]` are.

Obviously, the real option is http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-rdgGrp.html but we do not have enough encoding to support this (We miss many information and it would need manual tagging most of the time to decide the manuscript and so on...)

**Comment by lcerrato** [comment]
@PonteIneptique Yes, I understand the limitations of EpiDoc. Their distinction between ancient and modern hand is fairly clear. So if we are deviating from EpiDoc in favor of the TEI, that needs to be made clearer in our doc. 
Not advocating for one or the other, but my training was to follow EpiDoc as closely as possible.
